docs/guides/hiding_dashboards_from_sidebar.md

```diff
@@ -16,4 +16,4 @@ Rails.application.routes.draw do
 end
 ```
 
-In this case, only Orders and Products will appear in the sidebar, while Line Items can still appear as an association.
+In this case, only `Orders` and `Products` will appear in the sidebar, while `Line Items` can still appear as an association.
```

Comment:
Nitpick: these are not class names (eg: the class is `Order`, not `Orders`) so I'm not sure about formatting them as code. How about using double quotes instead?

```suggestion
In this case, only "Orders" and "Products" will appear in the sidebar, while "Line Items" can still appear as an association.
```